cli/popper/commands/cmd_stages.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+
+import click
+import popper.utils as pu
+
+from popper.cli import pass_context
+
+@click.command(
+    'stages', short_help='See and manipulate stages of a pipeline.'
+)
+@click.argument('pipeline', required=True)
+@click.option(
+    '--set',
+    help='Provide comma separated list of stages to set the stages of a '
```

Comment:
we can simplify and leave it at `Comma separated list of stages in the pipeline`.